egs/public_dataset/kiritan/run.sh

```diff
@@ -6,9 +6,11 @@
 . ./cmd.sh || exit 1;
 
 
-stage=0
-stop_stage=100
+stage=2
```

Comment:
change it to 0 and 100 as an initial stage setting

Reply:
ok